scripts/core/adapter.yml

```
@@ -200,3 +200,76 @@ etors:
     - name: NATIVE_CPU
       value: "5"
       desc: "The backend is Native CPU"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Minimum level of messages to be processed by the logger."
+class: $x
```

These should all be in the `$xAdapter` class.


```suggestion
class: $xAdapter
```